<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyOrderDialog</class>
 <widget class="QDialog" name="MyOrderDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>299</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="menuWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>450</width>
       <height>40</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>450</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">color:#FF6700</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QPushButton" name="allOrderBtn">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;color:#333;border:none;padding:4px 12px;</string>
        </property>
        <property name="text">
         <string>全部订单</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="upcomingOrderBtn">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;color:#333;border:none;padding:4px 12px;</string>
        </property>
        <property name="text">
         <string>待出行</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="finishedOrderBtn">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;color:#333;border:none;padding:4px 12px;</string>
        </property>
        <property name="text">
         <string>已完成</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="cancelledOrderBtn">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>12</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;color:#333;border:none;padding:4px 12px;</string>
        </property>
        <property name="text">
         <string>已取消</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="orderTable">
     <property name="editTriggers">
      <set>QAbstractItemView::EditTrigger::NoEditTriggers</set>
     </property>
     <property name="columnCount">
      <number>6</number>
     </property>
     <column>
      <property name="text">
       <string>订单号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>航班号</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>出发时间</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>目的地</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>状态</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>操作</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
